pages/glossary/owned-by.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Owned by
+key: owned-by
+---
+
+An element A is owned by element B if element A is a child of element B in the [accessibility tree](https://www.w3.org/TR/act-rules-aspects/#input-aspects-accessibility).
```

Comment:
Using "child" instead of "descendant" means that an example like:
```html
<ul>
  <div>
    <li>Foo</li>
  </div>
</ul>
```
is failing the rule because the `li` is only owned by the `div` which does not have the correct role.

Is this voluntarily (I guess so given that it's essentially Failed Example 3)? I am a bit afraid that situations like that can be fairly common with frameworks adding a lot of intermediate elements all over the place.

Reply:
You're sorta right, but for the wrong reason. In Firefox that ul > div > li is a failure, because the div will be included as a container in the accessibility tree. To fix this, authors need to put role=presentation on that div. In Chrome this isn't a failure, because Chrome's accessibility tree treats divs as null roles (not containers), and null roles are not included in the accessibility Tree.

That's why we changed this rule to test the accessibility tree instead of the DOM. The differences between various accessibility trees are too substantial, and there is no standard to refer to, for us to be able to say what is right and what isn't. So instead, the appropriate thing to do is to just say: whatever accessibility tree you have, a listitem must be a child of a list. Because that's in the spec and things don't work if you don't do that.